File: CS50_Coursework/sentimental-mario-less/mario.py
from cs50 import get_int


def main():
    height = get_height()
    for i in range(height):
        print((" " * (height - i - 1)) + ("#" * (i + 1)), end="")
        print()


def get_height():
    while True:
        n = get_int("Height: ")  # or int(input("Height: "))
        if n > 0 and n < 9:
            return n
        # try: except: else:  blocks->look at scores.py


main()

File: CS50_Coursework/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    while (height < 1 || height > 8)
    {
        height = get_int("Height: ");
    }

    for (int i = 1; i <= height; i++)
    {
        int space = height - i;
        for (int j = 0; j < space; j++)
        {
            printf(" ");
        }
        for (int h = 0; h < i; h++)
        {
            printf("#");
        }
        printf("  ");
        for (int h = 0; h < i; h++)
        {
            printf("#");
        }
        printf("\n");
    }

}